.hooks/bootstrap

```diff
@@ -76,6 +76,12 @@ else
             echo "ERROR: virtualenv is missing. Please install using: pip install virtualenv" 1>&2
             exit 1
         fi
+        VIRTUALENV_VERSION=$(virtualenv --version)
+        if [[ $VIRTUALENV_VERSION != *"16.7.7"* ]]; then
```

Comment:
I think any version of 16.7.* is good. 
```suggestion
        if [[ $VIRTUALENV_VERSION != *16.7.* ]]; then
```

Reply:
Also this check needs to be done only if we are going to install PY2. So it should be done later on in the code and check what we concluded regarding PY2.

Reply:
I added verification